### PR TITLE
Ad upport for odoo 9

### PR DESCRIPTION
**Pull request by dansanti**
- This PR make it working in odoo 9.
- Is necessary some revision if are there any unused method on new api.

### CODE REVIEW

File: pos_product_template/__openerp__.py [hunk 1]
```diff
@@ -75,5 +75,5 @@
     'images': [
         'static/src/img/screenshots/pos_product_template.png',
     ],
-    'installable': False,
+    'installable': True,
```
Comment (elicoidal) [inline pos_product_template/__openerp__.py:78]: @dansanti   Thanks for your contribution to the OCA. 
There is a little more than this key to change in openerp.py.
Let me introduce to you some basic information about contribution in OCA and I will review other points in more comments inside the code..
In general, please follow the principles: 
- Contribution guidelines: https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md
- Please follow the OCA README template:
  https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst
- We recommend to include a screenshot as well to help new comers (nice to have)
- Please use simple headers in all py files following OCA templates
  https://github.com/OCA/maintainer-tools/blob/master/template/module/__openerp__.py#L3
- Please sort all imports in alphabetical order following: https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#imports

File: pos_product_template/static/src/js/ppt.js [hunk 2]
```diff
@@ -17,163 +17,141 @@
     along with this program.  If not, see <http://www.gnu.org/licenses/>.
```
Comment (elicoidal) [inline pos_product_template/static/src/js/ppt.js:17]: Please simplify headers following newest templates

**Comment by rafaelbn** [comment]
Hi @dansanti are you going to finish this PR? You have to rebase and attend @elicoidal comments, thanks.

**Comment by rafaelbn** [comment]
I found same PR here: https://github.com/OCA/pos/pull/97

**Comment by legalsylvain** [comment]
closing duplicated PR.

**Pull request closed without merging**